master/custom/workers.py

```diff
@@ -23,7 +24,10 @@ def __init__(self, settings, name,
         pw = worker_settings.get('password', None) or owner_settings.password
         owner_email = owner_settings.get('email', None)
         emails = list(map(str, filter(None, (settings.get('status_email', None), owner_email))))
-        self.bb_worker = _worker.Worker(name, str(pw), notify_on_missing=emails)
+        if settings.value == ...:
+            self.bb_worker = _worker.LocalWorker(name)
+        else:
+            self.bb_worker = _worker.Worker(name, str(pw), notify_on_missing=emails)
```

Comment:
You can also just run whichever worker(s) you want separately rather than trying to run them all locally.

Reply:
>You can also just run whichever worker(s) you want separately rather than trying to run them all locally.

Hummmm, you mean create the worker process separately? I find that a bit cumbersome, but if you think it makes more sense, we can do that.

Reply:
I'm just afraid that replacing `Worker` with `LocalWorker` means that it's going to try to run *all* of our defined workers locally, which is just going to lead to headache :).  But I'm perfectly fine with it being a (non-default) option.

Reply:
> I'm just afraid that replacing `Worker` with `LocalWorker` means that it's going to try to run _all_ of our defined workers locally, which is just going to lead to headache :). But I'm perfectly fine with it being a (non-default) option.

Hum, maybe the correct thing to do is factor out the builders (because they have hardcoded worker names) into another module and create a `ExampleBuilder`. What do you think?

Reply:
I think what you've done with the `Test Builder` and `local-worker` are fine.